_docs/overview/design-getting-started.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Getting Started
+layout: docs
+category: Overview
+usage: >
+  Read this article to learn how to use the Helix design language, which will help you design faster and provide a more consistent user experience.
+status: stable
+resource: true
+last-modified: 2017-10-24
+---
+
+
+## Prerequisites
+
+<div class="hxRow large">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+To take full advantage of Helix resources, you'll need Sketch. [Download Sketch here](http://www.sketchapp.com). Learn more about using Sketch with [these Pablo video lectures](https://www.youtube.com/watch?v=cEplnCnZuDM&list=PLWlUJU11tp4fEXI8deWhBQAHDv9R23WHB).
+
+{% endcolumn %}
+
+</div>
+
+
+## Get in the loop
+
+<div class="hxRow large">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+Join the following channels to talk to the Helix team and get updates.
+
+- Join the [#helix Slack channel](https://rackspace.slack.com/messages/#helix)
+- [Sign up for email updates](mailto:helix@rackspace.com)
+- Follow the [Helix Github Repo](https://github.com/rackerlabs/design-system)
+
+{% endcolumn %}
+
+</div>
+
+## Learn the language
+
+<div class="hxRow large">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+Familiarize yourself with the concepts that define the Helix language. Through this site, you can see how they appear in the layouts, styles, and componenets used in Rackspace UIs. 
+
+- **Principles**: The ideas and guidelines for designing to the Helix language.
```

Comment:
Suggestion: 
`Principles: Ideas and guidelines for making good, consistent decisions when designing in Helix`

Reply:
+1, end with full stop.

Reply:
updated.